CHANGELOG.md

```diff
@@ -10,10 +10,12 @@ All notable changes to this project will be documented in this file.
 ￼ config property `requestedSecretLifetime`. This helps reducing frequent Pod restarts ([#660]).
 - Run a `containerdebug` process in the background of each "druid" container to collect debugging information ([#667]).
 - Aggregate emitted Kubernetes events on the CustomResources ([#677]).
+- Support Apache Druid `31.0.1` and adjust memory limits of coordinator and middlemanager, remove `26.0.0` ([#685]).
```

Comment:
Can you please split that into two release note entries and mark the memory change as breaking? Reason is that existing namespaces/clusters can run out of memory and disturb availability.
As well as please note the default was changed from x to y.